src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,57 @@
+package calculator;
+
+import java.util.Stack;
+
+public class Calculator {
+    private static final Stack<String> stack = new Stack<>();
+    public static final String INPUT_SPLITTING_REGEX = " ";
+    public static final String ERROR_MESSAGE_ON_INVALID_INPUT = "입력값이 없습니다.";
+    public static final String NUMEIRC_REGEX = "\\d+";
+    public static final String OPERATOR_REGEX = "[+\\-*/]";
+
+    public static int calculate(String input) {
+        checkNotEmpty(input);
+        for (String token : input.split(INPUT_SPLITTING_REGEX)) {
+            handleNumberToken(token);
+            handleOperatorToken(token);
+        }
+        return Integer.parseInt(stack.pop());
+    }
+
+    private static void checkNotEmpty(String input) {
+        if (input.isEmpty()) {
+            throw new IllegalArgumentException(ERROR_MESSAGE_ON_INVALID_INPUT);
+        }
+    }
+
+
+    public static int operate(String notation, int left, int right) {
+        Operator operator = Operator.fromNotation(notation);
+        return operator.action.apply(left, right);
```

Comment:
잘 수정 해주셨네요 💯 
질문주신 부분에 대해선 아래처럼 사용해보면 어떨까요? 

```suggestion
        return operator.operate(left, right);
```

```
    // Operator.java 

    public int operate(int left, int right) {
        return this.action.apply(left, right);
    }
```